config/betamocks/services_config.yml

```diff
@@ -1,6 +1,28 @@
 ---
 :services:
 
+- :name: 'MHV_Rx'
+  :base_uri: <%= "#{URI(Settings.mhv.rx.host).host}" %>:443
```

Comment:
Most services deduce the port based on the URI.  For example with EVSS:
```ruby
<%= "#{URI(Settings.evss.url).host}:#{URI(Settings.evss.url).port}" %>
```
I'm not even sure if it's better to be honest, but it is precedent.  Just making you aware, but not a show stopper.